### PR TITLE
RFC 68: Make local imports in Python file handlers root-relative

### PR DESCRIPTION
**Pull request by stephenmcgruer**
Implementation CL: https://github.com/web-platform-tests/wpt/pull/26328/

### CODE REVIEW

File: rfcs/python-file-handler-local-imports.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# RFC ??: Make local imports in Python file handlers root-relative
```
Comment (stephenmcgruer) [inline rfcs/python-file-handler-local-imports.md:1]:
```suggestion
# RFC 68: Make local imports in Python file handlers root-relative
```

**Review by jgraham — APPROVED** [review]
This isn't lovely, but we haven't found a better alternative so far.

**Review by Hexcles — APPROVED** [review]
LGTM

**Review by Ms2ger — APPROVED** [review]
A little annoying, but looking forward to the py3-only future.

**Pull request merged**